truss/templates/trtllm/packages/constants.py

```diff
@@ -1,7 +1,9 @@
 from pathlib import Path
 
 # If changing model repo path, please updated inside tensorrt_llm config.pbtxt as well
-TENSORRT_LLM_MODEL_REPOSITORY_PATH = Path("/packages/tensorrt_llm_model_repository/")
+TENSORRT_LLM_MODEL_REPOSITORY_PATH = Path(
+    "/app/packages/tensorrt_llm_model_repository/"
+)
```

Comment:
An analogous change would be needed in the whole truss-example repo.

Reply:
I don't think truss examples reference the packages in the docker file directly, only the one in the truss directory which doesn't need to change. The only reason this one changes in that it's directly templating into the docker image, which truss examples can't do directly. I'll take a pass through them though to see if its needed.

Reply:
actually, you're right (I didn't know this was happening). This TRT-LLM truss is using it: https://github.com/basetenlabs/truss-examples/blob/f7295157fb38826afdbf7c59fcf59e8882900645/templates/trt-llm/packages/utils.py#L25

@aspctu do you envision migrating that truss-example to the new TRT-LLM templating in truss soon or should I make sure to coordinate the rollout with truss-examples?

Reply:
That `truss-examples` template is going to be removed, we don't use that anymore and have disabled it for the past couple weeks. We'll be moving towards a world where the template lives inside `truss` so feel free to ignore that.